src/main/java/racing/RacingMain.java

```diff
@@ -0,0 +1,29 @@
+package racing;
+
+import racing.domain.Car;
+import racing.domain.Judgement;
+import racing.domain.RacingGame;
+import racing.views.InputView;
+import racing.views.ResultView;
+
+import java.util.List;
+
+public class RacingMain {
+
+    public static void main(String[] args) {
+
+        String[] carNames = InputView.inputCarNames();
+
+        RacingGame racingGame = new RacingGame(carNames);
+
+        int tryTimes = InputView.inputTryTimes();
+
+        ResultView.showResult();
+
+        for (int j = 0; j < tryTimes; j++) {
+            racingGame.moveCars();
+        }
+
+        ResultView.printWinner(new Judgement(racingGame.getCars()).getWinnerCars());
```

Comment:
```suggestion
        ResultView.printWinner(racingGame.getwWinnerCars());
```
현재는 `getCars()`로 직접 꺼내서 처리하고 있는데, `RacingGame` 내부에서 판단하면 어떨까요?
`RacingGame` 내에서 `Judgement`를 활용해 우승자를 직접 가져오도록 변경하면 캡슐화를 더 잘 지킬 수 있을 것 같아요!